client/console_client.go

```diff
@@ -279,11 +282,15 @@ func (client *Client) Apply(resource *resource.Resource, dryMode bool) (string,
 	return upsertResponse.UpsertResult, nil
 }
 
-func (client *Client) Get(kind *schema.Kind, parentPathValue []string, queryParams map[string]string) ([]resource.Resource, error) {
+func (client *Client) Get(kind *schema.Kind, parentPathValue []string, parentQueryValue []string, queryParams map[string]string) ([]resource.Resource, error) {
```

Comment:
I think here (and other place), we should use a map[string]string and potentially merge with queryParams.
And therefore in cmd package build a map and not an array. It will IMO make the code easier to read

Reply:
Will do in another PR